packages/@aws-cdk/aws-codebuild/package.json

```diff
@@ -199,6 +199,8 @@
       "docs-public-apis:@aws-cdk/aws-codebuild.CodeCommitSourceProps.repository",
       "props-default-doc:@aws-cdk/aws-codebuild.CodeCommitSourceProps.cloneDepth",
       "docs-public-apis:@aws-cdk/aws-codebuild.CommonProjectProps",
+      "docs-public-apis:@aws-cdk/aws-codebuild.CommonStartBuildOptions",
+      "docs-public-apis:@aws-cdk/aws-codebuild.StartBuildOptions",
```

Comment:
Add these docs please, it shouldn't be hard 🙂 (And if I'm right about `CommonStartBuildOptions`, you'll only have to do it for `StartBuildOptions`!).